content/docs/command-reference/fetch.md

```diff
@@ -73,43 +71,43 @@ or `-T` options are used).
 
 ## Options
 
-- `-r <name>`, `--remote <name>` - name of the
-  [remote storage](/doc/command-reference/remote) to fetch from (see
-  `dvc remote list`).
+- `-r <name>`, `--remote <name>`- name of the
+  [remote storage](/doc/command-reference/remote) to fetch from
+  (see`dvc remote list`).
 
 - `--run-cache` - downloads all available history of stage runs from the remote
   repository.
 
-- `-d`, `--with-deps` - determines files to download by tracking dependencies to
-  the `targets`. If none are provided, this option is ignored. By traversing all
+- `-d`, `--with-deps`- determines files to download by tracking dependencies to
+  the`targets`. If none are provided, this option is ignored. By traversing all
   stage dependencies, DVC searches backward from the target stages in the
   corresponding pipelines. This means DVC will not fetch files referenced in
-  later stages than the `targets`.
+  later stages than the`targets`.
 
-- `-R`, `--recursive` - determines the files to fetch by searching each target
-  directory and its subdirectories for `dvc.yaml` and `.dvc` files to inspect.
-  If there are no directories among the `targets`, this option is ignored.
+- `-R`, `--recursive`- determines the files to fetch by searching each target
+  directory and its subdirectories for`dvc.yaml`and`.dvc`files to inspect. If
+  there are no directories among the`targets`, this option is ignored.
```

Comment:
Linter?

Reply:
I have no idea what happened here! Glad you noticed though... Fixed in b01ccd95 as well.

Reply:
Will keep an eye on this...